week0/homework/Person.js

```diff
@@ -0,0 +1,20 @@
+module.exports =  function (name,age)  {
```

Comment:
Of course functionally in JS, it's the same to use either an Function, or an Object with constructor or a class, but you should try to become more familiar with using the class keyword as that's going to be the preferred way going forward (and hence added as a new feature in ES spec).